src/runtime/api-test.js

```diff
@@ -244,20 +244,35 @@ describe('PurchaseData', () => {
 
 
 describes.sandboxed('SubscribeResponse', {}, () => {
-  let sr, pd, ud, complete, promise;
+  let sr, pd, ud, ent, complete, promise;
```

Comment:
all these variables have very unclear names to people like me who know nothing about this. What is sr, pd, ud, ent?

Reply:
ah now I see. I'd rename these to have the complete name

Reply:
Done